catalog-info.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: backstage.io/v1alpha1
+# This is the name of the component that will show up in the sidebar
+kind: Component
+metadata:
+  name: KotlinAudio
+  title: kotlinaudio
+  description: The repo that supports the submodule which is a fork of the original KotlinAudio but allows us to use reflection to integrate Conviva
+  annotations:
+    backstage.io/techdocs-ref: dir:.
+    # circleci.com/project-slug: NA 
+    # newrelic.com/dashboard-guid: NA
+    # snyk.io/org-name: NA
+    # snyk.io/project-ids: NA
+spec:
+  type: service
+  owner: group:the-economist/core-audio
+  lifecycle: Production
+   #system: NA
```

Comment:
perhaps this can be 
`system: system:the-economist/lamarr-core-app`

Reply:
done